help/datagrok/solutions/domains/bio/bio.md

```diff
@@ -302,14 +303,53 @@ To learn more about filtering, watch this [video](https://www.youtube.com/watch?
 
 ## Manage monomer libraries
 
-The default [HELM monomer library](https://github.com/datagrok-ai/public/blob/master/packages/Bio/files/monomer-libraries/HELMCoreLibrary.json) is pre-installed with the [Bio package](https://github.com/datagrok-ai/public/tree/master/packages/Bio). You can add your own monomer libraries using the dialog accessible from **Top Menu** > **Bio** > **Manage** > **Monomer Libraries**:
+Datagrok allows you to manage monomer libraries for any Macromolecule type (DNA, RNA, Peptide or custom). Monomer libraries are used to define the monomers, their properties and molecular structures that are
```

Comment:
Probably need to expand a bit more on why this is needed, such as (just listing the ideas, they need to be worded better)
* manage building blocks for polymers
* manage access privileges (this one is coming)
* manage arbitrary metadata on the monomer label (that we can use for instance for choosing which monomers to use for enumeration in the PolyTool)